newlib/libc/machine/riscv/memcpy-asm.S

```diff
@@ -28,5 +28,150 @@ memcpy:
 2:
   ret
 
-  .size	memcpy, .-memcpy
+#else
+    mv a3, a0
```

Comment:
the alignment of the operands is strange here. try and have it consistent within the file.